config/peerpods/podvm/aws-podvm-image-handler.sh

```diff
@@ -97,7 +97,7 @@ function install_aws_cli() {
         error_exit "Failed to download aws cli"
 
     # Install aws cli v2
-    unzip -o "/tmp/awscliv2.zip" -d "/tmp" ||
+    unzip -q -o "/tmp/awscliv2.zip" -d "/tmp" ||
```

Comment:
/lgtm
This can also be clubbed in a separate PR along with first two commits and merged.